src/mscorlib/Resources/Strings.resx

```diff
@@ -3596,4 +3654,13 @@
   <data name="SynchronizationLockException_MisMatchedWrite" xml:space="preserve">
     <value>The write lock is being released without being held.</value>
   </data>
-</root>
+  <data name="Memory_ThrowIfDisposed" xml:space="preserve">
+    <value>The memory has been disposed.</value>
```

Comment:
Nit: it's a bit odd to me to see "memory" here in lower case when talking about disposing of it.  It'd make more sense to me if it were either `The Memory<T> has been disposed.` or `The memory has been freed.`, but the combination just strikes me weirdly.